pkg/rbd/controllerserver.go

```diff
@@ -626,3 +626,8 @@ func (cs *ControllerServer) ListSnapshots(ctx context.Context, req *csi.ListSnap
 		Entries: entries,
 	}, nil
 }
+
+func (cs *ControllerServer) ControllerExpandVolume(ctx context.Context, req *csi.ControllerExpandVolumeRequest) (*csi.ControllerExpandVolumeResponse, error) {
```

Comment:
afaict we do not need these, as the default files already have the required definitions.